api/api/api_exception.py

```diff
@@ -34,7 +34,9 @@ def __init__(self, code: int, details: str = None):
             2008: 'Experimental features are disabled. '
                   'It can be changed in the API configuration',
             2009: 'Semicolon (;) is a reserved character and must '
-                  'be percent-encoded (%3B) to use it.'
+                  'be percent-encoded (%3B) to use it.',
+            2010: f'Error loading max_memory_usage. Minimum value '
+                  f'must be {MIN_VALUE_MAX_MEMORY_USAGE}'
```

Comment:
Also, add an indication to explain that the value could be `0` so the memory usage is not limited.

```suggestion
            2010: f'Error loading max_memory_usage. Value must be 0 (limitless) '
                  f'or greater than {MIN_VALUE_MAX_MEMORY_USAGE}.'
```